lib/check-sanity.js

```diff
@@ -0,0 +1,21 @@
+export default function (point) {
+  checkCoord(point.x);
+  checkCoord(point.y);
+}
+function checkCoord(num) {
+  if (typeof Number.isFininte === 'function') {
+    if (Number.isFininte(num)) {
+      return;
+    }
+    throw new TypeError('coordinates must be finite numbers');
+  }
+  if (typeof num !== 'number') {
+    throw new TypeError('coordinates must be finite numbers');
+  }
+  if (num !== num) {
+    throw new TypeError('coordinates must be finite numbers');
+  }
+  if (!isFinite(num)) {
```

Comment:
well if the newer one doesn't exist, we have to do 3 checks to get the same functionality including this one